src/components/Icon/Icon.test.tsx

```diff
@@ -59,20 +58,4 @@ describe('<Icon />', () => {
       });
     });
   });
-
-  describe('each Icon', () => {
-    let Icon: any;
-
-    // tslint:disable-next-line:mocha-no-side-effect-code
-    Object.keys(allExportedIcons).forEach(name => {
-      beforeEach(() => {
-        Icon = (allExportedIcons as any)[name];
-        component = shallow(<Icon />);
-      });
-
-      it('has a 240x240 viewbox', () => {
-        expect(component.getElement().props.viewBox).toBe('0 0 240 240');
```

Comment:
I deleted this for now because the change broke it, but think I'll probably have to fix it and add it back for coverage.